src/client/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

```diff
@@ -57,7 +57,7 @@ public abstract class ApplicationOptions
         /// This redirect URI needs to be registered in the app registration (https://aka.ms/msal-net-register-app).
         /// In MSAL.NET, <c>IPublicClientApplication</c> defines the following default RedirectUri values:
         /// <list type="bullet">
-        /// <item><description><c>urn:ietf:wg:oauth:2.0:oob</c> for desktop (.NET Framework and .NET Core) applications</description></item>
+        /// <item><description><c>https://login.microsoftonline.com/common/oauth2/nativeclient</c> for desktop (.NET Framework and .NET Core) applications</description></item>
```

Comment:
I still think these chagnes need to be in a separate commit.

Reply:
i missed this one